addons/mail/models/mail_channel.py

```diff
@@ -1153,6 +1158,28 @@ def channel_create(self, name, privacy='public'):
         self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), channel_info)
         return channel_info
 
+    @api.model
+    def group_chat_create(self, partners_to):
+        """ Create a private group chat with at least 3 partners.
```

Comment:
This mention about "3" is actually not correct, it's perfectly fine to create a group chat with one person alone, or 2 persons as well. Other participant might come at a later time thanks to invite link and such.

Group chats should never be defined or limited by the number of participants (except maybe for an upper bound if necessary).